pom.xml

```diff
@@ -212,9 +212,9 @@
         <module>extensions-contrib/aliyun-oss-extensions</module>
         <module>extensions-contrib/prometheus-emitter</module>
         <module>extensions-contrib/opentelemetry-emitter</module>
+        <module>extensions-contrib/opentelemetry-extensions</module>
```

Comment:
Reason for this move:

[Semaphore build jobs](https://confluentinc.semaphoreci.com/jobs/be6e5d39-97f1-4ab3-9a9e-03f1b8ec78fe) were failing with the following error:

```
[ERROR] Failed to execute goal de.thetaphi:forbiddenapis:3.2:check (default-cli) on project druid-opencensus-extensions: 
Check for forbidden API calls failed while scanning class 'org.apache.druid.data.input.opencensus.protobuf.OpenCensusProtobufInputRowParser' 
(OpenCensusProtobufInputRowParser.java): java.lang.ClassNotFoundException: org.apache.druid.data.input.opentelemetry.protobuf.AbstractProtobufReader (while looking up details about referenced class 'org.apache.druid.data.input.opencensus.protobuf.OpenCensusProtobufReader') -> [Help 1]
```

It looked like the `forbiddenapis` check was failing because of some classpath issue, even though the compilation step was passing.   I wasn't able to reproduce this failure consistently on my laptop (but failed every time in Semaphore)

[This issue](https://github.com/policeman-tools/forbidden-apis/issues/109) ***looks*** to be closely related. 


I initially tried using the `@SuppressForbidden` annotation for the `OpenCensusProtobufInputRowParser` class and that led to the same error for the `OpenCensusProtobufReader` class. 
Ultimately [Semaphore jobs passed](https://confluentinc.semaphoreci.com/workflows/8b724c26-9094-4c76-81f5-e7dae520a9a0?pipeline_id=93f04baa-acdd-4f26-a324-97411f07aaed) with this change. I'm not sure how this change in module ordering fixes up the `forbiddenapis` check.